libraries/radspreadprocessing/changes-and-backward-compatibility/backward-compatibility.md

```diff
@@ -13,6 +13,16 @@ position: 1
 
 This article will list the breaking changes and how they can be fixed when upgrading from a specific version of the controls to the next one.
 
+## What's Different in 2022 R3
+
+### Changed
+
+The __UpdateConditionalFormattingCellRanges__ method is marked obsolete.
+
+### What to do now
+
+Use use the combination of __RemoveConditionalFormatting__ and __AddConditionalFormatting instead__.
```

Comment:
Use combination of __RemoveConditionalFormatting__ and __AddConditionalFormatting__ instead__.

Reply:
Fixed